tools/reviewer/web/src/app/dashboard/reviews-panel/reviews-panel.component.ts

```diff
@@ -31,6 +28,10 @@ export class ReviewsPanelComponent implements OnInit {
         if (user) {
           const email = user.email;
           this.username = email.split('@')[0];
+          // TODO: Maybe to use email instead of username?
```

Comment:
Why would it be better?

Reply:
I based my opinion on an idea, that there could be different domains. E.g.:
oferb@gmail.com
oferb@fakemail.com

But now I realized, that it's always gmail.com. (and maybe google.com..)
So yeah, username is shorter.